<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageHome</class>
 <widget class="QWidget" name="PageHome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>690</height>
   </rect>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QGridLayout" name="layPanels">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="pageTopLeft" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="layoutWidget1">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>371</width>
           <height>336</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="layTopLeft">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="spacing">
           <number>12</number>
          </property>
          <item row="3" column="1">
           <widget class="SpinBox" name="sbxFixed2">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblClosed">
            <property name="text">
             <string>Closed</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="SpinBox" name="sbxFixed3">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblFixed2">
            <property name="text">
             <string>Fixed #2</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblFixed3">
            <property name="text">
             <string>Fixed #3</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QToolButton" name="tbnSetFixed3">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="tbnSetClosed">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lblOpen">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblFixed1">
            <property name="text">
             <string>Fixed #1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="SpinBox" name="sbxFixed1">
            <property name="maximum">
             <number>127</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="SpinBox" name="sbxOpen">
            <property name="maximum">
             <number>1023</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="tbnSetFixed1">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="SpinBox" name="sbxClosed">
            <property name="maximum">
             <number>1023</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="tbnSetOpen">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QToolButton" name="tbnSetFixed2">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="pageBottomRight" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>371</width>
           <height>337</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="layBottomRight">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="2" column="1">
           <widget class="QToolButton" name="tbnSetAsClosed">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Set As Closed</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="tbnSetAsOpen">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Set As Open</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="lblHHLevel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="tbnRestore">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Restore</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/restore.png</normaloff>:/icons/restore.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QToolButton" name="tbnSaveAll">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save All</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/microchip.png</normaloff>:/icons/microchip.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QToolButton" name="tbnSetAll">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Set All</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="hsp">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>60</width>
              <height>48</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layHHLevelBar">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="LevelBar" name="pbrHHLevel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>1023</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LevelBar</class>
   <extends>QProgressBar</extends>
   <header>levelbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
